src/common/attachments/uploadAttachmentsDialog.component.test.tsx

```diff
@@ -96,18 +96,24 @@ describe('Upload attachment dialog', () => {
     // Checks if file extension is displayed. If it's editable, actual value will disappear after editing.
     expect(await screen.findByText('.txt')).toBeInTheDocument();
 
-    const title = screen.getByRole('textbox', { name: 'Title' });
-    const description = screen.getByRole('textbox', { name: 'Description' });
+    const description: HTMLInputElement = screen.getByRole('textbox', {
+      name: 'Description',
+    });
+
+    const title: HTMLInputElement = screen.getByRole('textbox', {
+      name: 'Title',
+    });
 
     expect(await screen.findByDisplayValue('test1')).toBeInTheDocument();
-    expect(await screen.findByText('.txt')).toBeInTheDocument();
 
-    await user.type(title, 'test title');
+    const delayedUser = userEvent.setup({ delay: 20 });
 
-    expect(await screen.findByDisplayValue('test title')).toBeInTheDocument();
-    await user.click(description);
+    await delayedUser.type(title, 'test title');
+
+    await delayedUser.type(description, 'test description');
+
+    expect(await screen.findByText('.txt')).toBeInTheDocument();
```

Comment:
Can you add a comment explaining what you think is happening and why you chose the fix. Also make sure to rerun the unit test a couple of times on the CI to make sure that the implementation has worked